### PR TITLE
Added a modified index.html along with all of the corresponding scripts and image

### PR DESCRIPTION
**Pull request by Jibarra72**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,22 +3,22 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>BackpackPacker</title>
+    <title>Forest</title>
     <link href="https://fonts.googleapis.com/css2?family=Oswald:wght@200..700&family=Work+Sans:wght@100..900&display=swap" rel="stylesheet" />
     <link rel="stylesheet" href="style.css" type="text/css" media="all" />
-    <script type="module" src="Backpack.js"></script>
-    <script type="module" src="script.js"></script>
+    <script type = "module" src = "script.js"></script>
```
Comment (nagarci7) [inline index.html]: there shouldn't be a space between the attributes, equal sign, and value. It should look like 
<script type="module" src="scripts.js"></script> Please update.

**Review by nagarci7 — CHANGES_REQUESTED** [review]
@Jibarra72 please make an update to lines 9 and 10 of the index.html file.

**Comment by Jibarra72** [comment]
I have updated the index.html as you requested

**Review by nagarci7 — APPROVED** [review]
@Jibarra72 looks great! it's good to merge

**Pull request merged**